public/spec/auth/spec.md

```diff
@@ -250,7 +250,7 @@ public client class ListenerLdapUserStoreBasicAuthHandler {
         self.provider = new (config);
     }
     
-    remote function authenticate(Request|Headers|string data) returns auth:UserDetails|Unauthorized {
+    remote method authenticate(Request|Headers|string data) returns auth:UserDetails|Unauthorized {
```

Comment:
All changes in this file seem to be changes to code snippets. Should remain as `function` in code snippets.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/6013.